charts/amazon-cloudwatch-observability/crds/cloudwatch.aws.amazon.com_dcgmexporters.yaml

```diff
@@ -1293,6 +1293,48 @@ spec:
                 description: TlsConfig is the raw YAML to be used as the exporter
                   TLS configuration.
                 type: string
+              tolerations:
+                description: Toleration to schedule OpenTelemetry Collector pods.
```

Comment:
Same as last time. These should say DCGM or Neuron accordingly instead of OpenTelemetry Collector pods.